src/components/adds/horizontal.tsx

```
@@ -0,0 +1,57 @@
+    };
+
+    fetchAdData();
+  }, []);
+
+  if (loading) return <div>Loading...</div>;
+  if (error) return <div>{error}</div>;
+
+  return (
+    <div>
+      {adData?.code ? (
+        <div dangerouslySetInnerHTML={{ __html: adData.code }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)